public/apps/configuration/configuration-app.tsx

```diff
@@ -21,14 +21,22 @@ import { I18nProvider } from '@osd/i18n/react';
 import { AppMountParameters, CoreStart } from '../../../../../src/core/public';
 import { SecurityPluginStartDependencies, ClientConfigType } from '../../types';
 import { AppRouter } from './app-router';
+import { DataSourceManagementPluginSetup } from '../../../../../src/plugins/data_source_management/public';
 
 export function renderApp(
   coreStart: CoreStart,
-  navigation: SecurityPluginStartDependencies,
+  depsStart: SecurityPluginStartDependencies,
```

Comment:
Nit: Consider an alternative name like `secStartupDeps` or something

Reply:
Since we are already in the security plugin code base, I think its fine to keep the name as depsStart. However in shimming this feature in, I did notice a lot of places where the naming could be improved, such as this parameter initially being called "navigation". I will resolve this comment for now, but let's open up a subsequent issue to take a look over the variable names once more. I think there was also some typos/mis-naming showing up on the user facing text that was brought up in other UX meetings.